ee/codegen/scripts/upgrade-workflow-server.mts

```diff
@@ -0,0 +1,30 @@
+import packageJson from "../package.json" assert { type: "json" };
+import { getGithubToken } from "./get-github-token.mjs";
+import { Octokit } from "@octokit/rest";
+
+/*
+ * This triggers the workflow server version release and lives here for convenience
+ * so it can take advantage of the existing github app auth token.
+ */
+const main = async () => {
+  const version = packageJson.version;
+  console.log("Upgrading workflow server to version", version);
+  const octokit = new Octokit({ auth: await getGithubToken() });
+  await octokit.actions.createWorkflowDispatch({
+    owner: "vellum-ai",
+    repo: "vembda-service",
+    workflow_id: "version-python-workflow-server.yaml",
```

Comment:
[oos] This is cool - part of me sorta wished I designed the codegen server bump like this to use an action in _that_ repo.